lib/easy.c

```diff
@@ -667,6 +667,7 @@ static CURLcode easy_transfer(struct Curl_multi *multi)
   while(!done && !mcode) {
     int still_running = 0;
     int rc;
+    long timeout_internal;
 
     before = Curl_now();
     mcode = curl_multi_wait(multi, NULL, 0, 1000, &rc);
```

Comment:
it is in `curl_multi_wait()`, the multi handle state is `CURLM_STATE_TOOFAST`. so, the variable rc is equal to zero.